packages/myst-frontmatter/src/licenses/licenses.spec.ts

```diff
@@ -27,12 +27,13 @@ describe('licenses are upto date with SPDX', () => {
         .sort((a, b) => a.licenseId.localeCompare(b.licenseId))
         .map((l) => [
           l.licenseId,
-          {
-            name: l.name,
-            osi: l.isOsiApproved || undefined,
-            free: l.isFsfLibre || undefined,
-            CC: l.licenseId.startsWith('CC') || undefined,
-          },
+          (() => {
+            const out: any = { name: l.name };
+            if (l.isOsiApproved) out.osi = true;
+            if (l.isFsfLibre) out.free = true;
+            if (l.licenseId.startsWith('CC')) out.CC = true;
+            return out;
+          })(),
```

Comment:
@fwkoch what is this change for?

Reply:
Maybe this was you debugging when the licenses failed? This looks fine and shouldn't pass if the licenses are out of date I don't think!